doc/source/methods/ProtoSelect.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<ol><li>The bullet list at the start isn't rendered properly (might need a space before the preceding sentence: https://github.com/SeldonIO/alibi/tree/master/doc#gotchas-when-writing-notebooks-as-examples)</li><li>the 3rd condition (is sparse) is confusing - a desirable prototype is 1 point but we talk here about it containing things (so the set P_l ?)</li><li>"integer program" - add link to wiki page? (https://en.wikipedia.org/wiki/Integer_programming)</li><li>Nitpick - are the labels (a) and (b) for the constraints necessary since we only refer to them as "first" and "second" constraint respectively? We could change labels to be (1) and (2) or change the text to refer to (a) and (b) for increased clarity. Also perhaps worth checking in we can add a space or right-justify the labels in the equation so they stand apart from the actual inequalities.</li><li>"can not" -&gt; "cannot" is better style :)</li><li>"proposed two alternative solution" -&gt; "propose two alternative solutions" (generally use present tense)</li><li>"&nbsp;The first one consist in a" -&gt; "The first one consists of" and "and transforming" -&gt; "and transformation of"</li><li>Please link to the Arxiv landing page for the paper, not directly to the pdf</li><li>Nitpick - I find it easier to read if not introducing terms like "state", e.g. "Given the current choice of prototype subsets (P_1, ..., P_l), in the next iteration we update it to (P_1, ...)"</li><li>Should we use display math for the Obj(x_j, l) ?</li><li>"given that" -&gt; "where" ? (since they are not constraints but definitions)</li><li>Same comments around labels (a) and (b) as above for the definitions of ksi, nu</li><li>Nitpick - perhaps using \lvert and \revert instead of \mid could give better typography</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<ol><li><br></li><li> You are correct. The properties refer to a prototype set. I updated.</li></ol>